components/cam/bld/namelist_files/use_cases/1850_cam5_av1c-04p2-MMF-1mom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
```

Comment:
We aren't actually using the 1850 use cases, are we?

Reply:
not at the moment, but it was an easy thing to add, should I delete it?

Reply:
removed